R/uvhydrograph-render.R

```diff
@@ -102,9 +102,10 @@ createSecondaryPlot <- function(data, month){
   table <- NULL
   status_msg <- NULL
   
-  isSecondarySeries <- any(grepl("downsampledSecondarySeries", names(data)))
+  isReferenceSeries <- any(grepl("downsampledReferenceSeries", names(data)))
+  isUpchainSeries <- any(grepl("downsampledUpchainSeries", names(data)))
```

Comment:
Yes, sometimes (eg: for discharge hydrographs), you'll get an upchain series and not a reference series. For others, you might get both, but alsojust the reference series.